units/provisioning_setup_host.go

```diff
@@ -928,11 +929,12 @@ func getMostRecentlyAddedDevice(ctx context.Context, env evergreen.Environment,
 		return blockDevice{}, errors.New("output contained no devices")
 	}
 
-	if !utility.StringSliceContains([]string{"", h.Distro.HomeDir()}, devices[len(devices)-1].MountPoint) {
+	lastDevice := devices[len(devices)-1]
```

Comment:
Could `len(devices)` ever be 0, leading to access out of bounds?

Reply:
That case is handled [here](https://github.com/evergreen-ci/evergreen/pull/3616/files#diff-b68d7c29013acf30e83cf282cd0f00feR928)
It seems unlikely, though, since there will always be at least a root volume (assuming we parsed the output correctly).